wp_api_integration_tests/tests/test_login_immut.rs

```diff
@@ -39,6 +39,7 @@ const VANILLA_WP_SITE_URL: &str = "https://vanilla.wpmt.co/wp-admin/authorize-ap
 #[case("vanilla.wpmt.co", VANILLA_WP_SITE_URL)]
 #[case("http://vanilla.wpmt.co", VANILLA_WP_SITE_URL)]
 #[case("https://optional-https.wpmt.co", "https://optional-https.wpmt.co/wp-admin/authorize-application.php")]
+#[case("https://gzip.wpmt.co", "https://gzip.wpmt.co/wp-admin/authorize-application.php")] // Returns a gzip-encoded response that the library should know how to deal with
```

Comment:
I could see an argument for not having this case (or not having it here) – I left it in there for now for discussion (since we should probably have it _somewhere_)